### PR TITLE
Hide password input when using key connector

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Hides the password input when using key connector.

Resolves https://app.asana.com/0/1201363553288890/1201394615545878
Depends on https://github.com/bitwarden/jslib/pull/560

## Screenshots

image

## Before you submit
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/app/accounts/lock.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <form id="lock-page" (ngSubmit)="submit()">
     <div class="content">
         <p aria-hidden="true"><i class="fa fa-lock fa-4x text-muted"></i></p>
-        <p>{{(pinLock ? 'yourVaultIsLockedPinCode' : 'yourVaultIsLocked') | i18n}}</p>
+        <p>{{'yourVaultIsLocked' | i18n}}</p>
```
Comment (djsmith85) [inline src/app/accounts/lock.component.html:4]: I think we should still show a different text when unlocking with PIN as it has no effect in regards to KeyConnector

**Review by djsmith85 — CHANGES_REQUESTED** [review]
If a user has biometrics enabled, they most likely would want to use it to log in. So if we move the biometrics button to the top, then we would only need to hide the submit button. This would greatly simplify the code and improve UX.

File: src/app/accounts/lock.component.html (hunk 1 above)
Comment (djsmith85) [reply src/app/accounts/lock.component.html:4]: @Hinton Spoke to Danielle and she is fine with only having one label. Do you think we could simplify the structure for the biometrics button with regards to my comment above? If not, it's looking good for approval

Comment (Hinton) [reply src/app/accounts/lock.component.html:4]: I think we could do that. Lets loop it by design.

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**